dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -24,7 +24,28 @@
               @change="handleChangeFrameworkEvent"
             />
             <slot name="reportingPeriodDropdown" />
+            <div class="flex align-content-start align-items-center pl-3">
+              <div
+                v-if="
+                  !isReviewableByCurrentUser &&
+                  dataType != DataTypeEnum.EutaxonomyNonFinancials &&
```

Comment:
we are going to implement the MLDT for eu taxo non financials and sme anyways in the near future, don't we?

Reply:
so this would only be useful for the time we take to implement MLDT? after that there will be no condition on the framework type... 
if that's the case, i dont think its worth it

Reply:
👍